src/biotite/sequence/profile.py

```diff
@@ -87,6 +87,20 @@ def __init__(self, symbols, gaps, alphabet):
         self._gaps = gaps
         self._alphabet = alphabet
 
+        if len(alphabet) != self.symbols.shape[1]:
+            raise ValueError(
+                f"The given alphabet is shorter ({len(self.alphabet)}) "
```

Comment:
It could also be the case, that the alphabet is longer than the number of columns.